pkg/component/observability/monitoring/kubestatemetrics/kubestatemetrics.go

```diff
@@ -32,6 +32,9 @@ const (
 
 	port            = 8080
 	portNameMetrics = "metrics"
+
+	suffixSeed    = "-seed"
+	suffixRuntime = "-runtime"
```

Comment:
Why not `-garden` if we already use `-seed`?

Reply:
To replace the garden-metrics-exporter in future by KSM's CRD monitoring, we see a demand to also have an instance to monitor the virtual-garden cluster. To do not make the name too long, I opted for just using the `-runtime` suffix instead of `-garden-runtime` and `-garden-virtual`. WDYT?

Reply:
Okay, agreed. Makes sense. Thank you!